fhir-persistence/src/main/java/com/ibm/fhir/persistence/payload/PayloadPersistenceHelper.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.persistence.payload;
+
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.zip.GZIPOutputStream;
+
+/**
+ * Collection of helper methods related to the persistence of FHIR resource payload data
+ */
+public class PayloadPersistenceHelper {
+    /**
+     * Compress the payload
+     * @param payload
+     * @return the compressed bytes
+     * @throws IOException
+     */
+    public static byte[] compressPayload(String payload) throws IOException {
```

Comment:
is this used?

Reply:
it should be used, this is the basis of a very common pattern I've seen in the code.